pkg/api/util/db.go

```diff
@@ -182,8 +182,19 @@ func CreateFileShareDBEntry(ctx *c.Context, in *model.FileShareSpec) (*model.Fil
 		return nil, errors.New(errMsg)
 	}
 
+	reg, err := regexp.Compile("^[a-zA-Z0-9_-]+$")
+	if err != nil {
+		errMsg := fmt.Sprintf("regex compilation for file name validation failed")
+		log.Error(errMsg)
+		return nil, errors.New(errMsg)
+	}
+	if reg.MatchString(in.Name) == false {
+		errMsg := fmt.Sprintf("invalid fileshare name it only contain english char and number  : %v", in.Name)
+		log.Error(errMsg)
+		return nil, errors.New(errMsg)
+	}
 	// validate the description
-	reg, err := regexp.Compile("[^a-zA-Z0-9 ]+")
+	reg, err = regexp.Compile("[^a-zA-Z0-9 ]+")
 	if err != nil {
```

Comment:
it's weird to validate description here

Reply:
@wisererik I have not added that code its already there. i just changed assignment type from := to = because i'm using same variable again. so := wont work here

Reply:
ok, please remove the description validation part.

Reply:
@wisererik, how about fixing it as an issue. This PR can be merged then

Reply:
OK